### PR TITLE
Added the gsoc 2023 blog

### LINKED ISSUE #87: Suggestion: Blog announcing GSOC 2023 interns
Are people interested in doing a blog announcing the Google Summer of Code Students for 2023, like we did last year?
https://www.pymc.io/blog/blog_gsoc_2022.html

We could update the title of the blog, such as 2023 Interns (instead of GSOC Summer 2023 Students), so:

**Meet our 2023 PyMC Interns**

### PR DESCRIPTION
**Pull request by error9098x**
Closes #87 
# Pull Request: Add 2023 PyMC GSoC Blog Introducing the Students

## Summary

This pull request adds a blog post to the [PyMC GSoC website](https://www.pymc.io/welcome.html) that introduces the students who are participating in the 2023 program. The post includes a brief biography of each student, as well as a link to their project proposal.

## Screenshots

Demo image

## Questions
* Are there any other improvements that could be made?
* Shreyas's blog link is not working

## References
* [PyMC GSoC 2022 Blog](https://www.pymc.io/blog/blog_gsoc_2022.html)

## Checklist

* The blog post is added to the `pymc.io/blog` directory.
* The blog post is written in Markdown format.
* The blog post is formatted correctly.
* The blog post includes a brief biography of each student.
* The blog post includes a link to each student's project proposal.

### CODE REVIEW

File: blog/blog_gsoc_2023.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+(gsoc_2023_presentations)=
+# Introducing the 2023 PyMC Google Summer of Code Students
+
```
Comment (OriolAbril) [inline blog/blog_gsoc_2023.md:3]: The post directive is missing, so it is not being listed in the blog section for the PR preview build.

Preview: https://pymcio--90.org.readthedocs.build/blog.html

Reference to post directive: https://github.com/pymc-devs/pymc.io/blob/main/blog/blog_gsoc_2022.md?plain=1#L4-L12. Not 100% sure about the metadata, @reshamas should probably weigh in on this.

**Review by OriolAbril — COMMENTED** [review]
Thanks for the PR, rendering wise everything looks great, even better than last years even as it is more consistent with indenting questions/answers and such. The metadata needs to be fixed though, see comment below.

On a more general note, probably for @reshamas, not sure if we are only planning a post for gsoc folks, if one for gsoc and one for non-gsoc or if it would make sense to include all summer interns in a single post. We got less slots than we asked for this year, so we are trying to find other funding for the interns we'd have liked to onboard but weren't able to. cc @fonnesbeck @ricardoV94

File: blog/blog_gsoc_2023.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+(gsoc_2023_presentations)=
+# Introducing the 2023 PyMC Google Summer of Code Students
```
Comment (reshamas) [inline blog/blog_gsoc_2023.md:2]:
```suggestion
# Meet our 2023 PyMC Interns

```

**Comment by reshamas** [comment]
> Thanks for the PR, rendering wise everything looks great, even better than last years even as it is more consistent with indenting questions/answers and such. The metadata needs to be fixed though, see comment below.
> 
> On a more general note, probably for @reshamas, not sure if we are only planning a post for gsoc folks, if one for gsoc and one for non-gsoc or if it would make sense to include all summer interns in a single post. We got less slots than we asked for this year, so we are trying to find other funding for the interns we'd have liked to onboard but weren't able to. cc @fonnesbeck @ricardoV94

@OriolAbril The title of the blog should be "Meet our 2023 PyMC Interns" as indicated in the issue description here:
https://github.com/pymc-devs/pymc.io/issues/87

File: blog/blog_gsoc_2023.md (hunk 2 above)
Comment (reshamas) [reply blog/blog_gsoc_2023.md:2]: @error9098x you can add your name as an author after "PyMC Interns"

**Comment by aloctavodia** [comment]
We should include Gabriel Stechschulte in this post he is working on this project https://github.com/pymc-devs/pymc/wiki/GSoC-2023-projects#better-tools-to-interpret-complex-regression-models

**Comment by reshamas** [comment]
@error9098x 
Thank you for this PR. 

@OriolAbril 
Added directive, but cannot see the blog in the preview. Not sure what else we are missing.

**Comment by reshamas** [comment]
> We should include Gabriel Stechschulte (@GStechschulte) in this post he is working on this project https://github.com/pymc-devs/pymc/wiki/GSoC-2023-projects#better-tools-to-interpret-complex-regression-models

@aloctavodia
Would you like to reach out to him and see if he wants to submit a PR with his info?
I did not see his name in the Slack.

He can also submit his info via "Commenting" in the Google doc

**Comment by GStechschulte** [comment]
> > We should include Gabriel Stechschulte (@GStechschulte) in this post he is working on this project https://github.com/pymc-devs/pymc/wiki/GSoC-2023-projects#better-tools-to-interpret-complex-regression-models
> 
> @aloctavodia Would you like to reach out to him and see if he wants to submit a PR with his info? I did not see his name in the Slack.
> 
> He can also submit his info via "Commenting" in the Google doc

Hey @reshamas and @aloctavodia, thanks for the tag. I will submit my information via the Google doc. Thanks!

**Comment by GStechschulte** [comment]
> > > We should include Gabriel Stechschulte (@GStechschulte) in this post he is working on this project https://github.com/pymc-devs/pymc/wiki/GSoC-2023-projects#better-tools-to-interpret-complex-regression-models
> > 
> > 
> > @aloctavodia Would you like to reach out to him and see if he wants to submit a PR with his info? I did not see his name in the Slack.
> > He can also submit his info via "Commenting" in the Google doc
> 
> Hey @reshamas and @aloctavodia, thanks for the tag. I will submit my information via the Google doc. Thanks!

This is done. Thanks!

**Comment by reshamas** [comment]
@error9098x 
Would you like to add in the info for the 3rd person we are featuring?

**Comment by error9098x** [comment]
Sorry for the late reply, actually my finals are going on, I will be only
able to do it on next Saturday/Sunday.

**Pull request merged**